udata/templates/organization/display.html

```diff
@@ -43,12 +43,12 @@
 {% block secondary_content %}
 {# Datasets and reuses tabs #}
 {% set dataset_tabs = (
-    ('datasets', _('Datasets'), datasets, {'organization': org.id|string}),
-    ('private-datasets', _('Private datasets'), private_datasets, {}),
+    ('datasets', ngettext('%(num)d dataset', '%(num)d datasets', datasets.total or 0), datasets, {'organization': org.id|string}),
+    ('private-datasets', ngettext('%(num)d private dataset', '%(num)d private datasets', private_datasets.total or 0), private_datasets, {}),
```

Comment:
Here, I discovered about the `private_datasets` concept. I was not if it was consistently populated so I assumed it had the same datastructure as `datasets`.